scroll-animations-1/Overview.bs

```diff
@@ -602,3 +653,154 @@ spec: cssom-view-1; type: dfn;
 	In such cases, the rendered scroll offset
 	and the state of a scroll-driven animation
 	may be inconsistent in the composited frame.
+
+# Appendix A: Named Timeline Ranges # {#timeline-ranges}
+
+	ISSUE: This section should move to CSS-ANIMATIONS-2 and WEB-ANIMATIONS-2.
+
+	This specification introduces the concept of [=named timeline ranges=]
+	to <a href="https://www.w3.org/TR/css-animations/">CSS Animations</a>
+	and <a href="https://www.w3.org/TR/web-animations/">Web Animations</a>.
+
+	A <dfn export>named timeline range</dfn>
+	is a named segment of an animation [=timeline=].
+	The start of the segment is represented as 0% progress through the range;
+	the end of the segment is represented as 100% progress through the range.
+	Multiple [=named timeline ranges=] can be attached to a given [=timeline=],
+	and multiple such ranges can overlap.
+	For example, the ''animation-timeline/contain'' range of a [=view progress timeline=]
+	overlaps with its ''animation-timeline/cover'' range.
+	[=Named timeline ranges=] are represented by
+	the <dfn><<timeline-range-name>></dfn> value type,
+	which indicates a [=CSS identifier=] representing
+	one of the predefined [=named timeline ranges=].
+
+	Note: In this specification, [=named timeline ranges=]
+	must be defined to exist by a specification
+	such as [[SCROLL-ANIMATIONS-1]].
+	A future level may introduce APIs for authors to declare
+	their own custom [=named timeline ranges=].
+
+## Named Timeline Range Keyframe Selectors ## {#named-range-keyframes}
+
+	[=Named timeline range=] names and percentages
+	can be used to attach keyframes
+	to specific progress points within the [=named timeline range=].
+	The CSS ''@keyframes'' rule is extended thus:
+
+	<pre class="prod">
+		<<keyframe-selector>> = from | to | <<percentage>> | <<timeline-range-name>> <<percentage>>
```

Comment:
Should these animation property changes go in the css-animations spec?

Reply:
Eventually, yes. Right now CSS Animations 2 is only an ED and it's not clear what the plan is for FPWD.